driver-sync/src/test/functional/com/mongodb/client/unified/EventMatcher.java

```diff
@@ -201,4 +201,5 @@ public static String getReasonString(final ConnectionClosedEvent.Reason reason)
                 throw new UnsupportedOperationException("Unsupported reason: " + reason);
         }
     }
+
```

Comment:
This new line (not at the end of file) is now the only change in the file. Let's roll it back.